engine/CMakeLists.txt

```diff
@@ -49,6 +49,12 @@ ELSE ()
 ENDIF ()
 
 
+# API Version for Game Assets
+# This is an incrementing number similar to the Google Android API Version
+# allowing us to differentiate our Assets API across multiple versions.
+# If a release is missing this value, then version `1` can be assumed.
+SET(VEGASTRIKE_ASSETS_API_VERSION "1")
```

Comment:
This line ended up in here twice. :-) Specifically on the 0.8.x branch.

Reply:
Darn missed that good catch @stephengtuggy .

Reply:
Probably from a mistake during cherry-picking.

Reply:
@stephengtuggy  filed #616 to fix this.